JakinChan200/CS180_Roald
# Jchan419 create pie chart

## Overview
Created a pie chart on the front page
Pie Chart displays the total amount of video data for each country


## Materials
- [#44](https://github.com/JakinChan200/CS180_Roald/issues/44) 

## Pictures
![image](https://user-images.githubusercontent.com/42459094/167204134-29144b3d-b1b5-4b13-a6e3-114e10a64a57.png)

## Manual Testing Guide
- Navigate to Home
- Wait 5 seconds
- You should see a pie chart on the bottom right of the page
